src/Core/src/Handlers/TimePicker/TimePickerHandler.Android.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using Android.App;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class TimePickerHandler : AbstractViewHandler<ITimePicker, MauiTimePicker>
+	{
+		MauiTimePicker? _timePicker;
+		AlertDialog? _dialog;
+
+		protected override MauiTimePicker CreateNativeView()
+		{
+			_timePicker = new MauiTimePicker(Context)
+			{
+				ShowPicker = ShowPickerDialog,
+				HidePicker = HidePickerDialog
+			};
+
+			return _timePicker;
+		}
+
+		protected override void DisconnectHandler(MauiTimePicker nativeView)
+		{
+			if (_dialog != null)
+			{
+				_dialog.Hide();
+				_dialog = null;
+			}
+		}
+
+		protected virtual TimePickerDialog CreateTimePickerDialog(int hour, int minute)
+		{
+			void onTimeSetCallback(object? obj, TimePickerDialog.TimeSetEventArgs args)
```

Comment:
nit: This is inconsistent with the show and hide. This is nested and they are not.

Reply:
Can you explain what you mean by that?

Reply:
The `onTimeSetCallback` method is nested. But no biggie.